### ISSUE TITLE
Feature Request(All in one device) 

### ISSUE THREAD

**Issue by macdog345** [QubesOS/qubes-issues#3967]
### Qubes OS version:R3.2

According to a reddit post there is no qubes for mobile in the works because of hardware support. So I 
just wanted to put this idea out there even though I doubt anyone will try to implement it(I might try one day though so please comment on feasibility). What if you had a very small laptop(ie the orwl) that you could easily fit in a purse or backpack or maybe some weird huge chest pocket lol(iron man core ofc) and it connected to a phone like device(just a monitor essentially). That way you could have a mobile qubes(for those of us who put in the effort of carrying it around) that doubled as a laptop when you brought it home, an all in one device. Over the top I know but I love the security of qubes and I want to carry it with me everywhere haha, I feel exposed entering my data in to my phone now =p. Also taking it further somehow connecting a phone radio through a usb converter and making it my phone 8).

Specifics: I guess the only things needed would be a qubes desktop that could work on any screen size, and a phone that was designed to pretend it was a monitor and touchpad, a photorpad. 

### Affected component(s): 

---

### General notes:

---

**Comment by andrewdavidwong** [comment]
I like the general idea. Lately, I've been thinking that it would be nice to have a pocket-sized x86 laptop that I can carry around everywhere as if it were a smartphone but that I would use just like a Qubes laptop. I recall someone on the MLs a while back asking about the smallest device that supports Qubes with a similar intent in mind.

Since this sounds like it's primarily a hardware issue and wouldn't necessarily involve any modification of Qubes itself (at least, no clear need known at this early stage), it doesn't seem like there's anything actionable for this as a qubes-issue (at least not yet). Therefore, I think it would be more appropriate to discuss this on qubes-devel.

**Comment by macdog345** [comment]
The google group qubes-devel right, should I repost it or is it so far in the future that I'll just leave it up to you =p? And specifically I was thinking qubes isn't optimized for a smaller screen(ie button/text size, an app launcher similar to phones, etc). But I guess without any supporting hardware it's not really worth it at this point like u said. I wonder if you could just take a regular phone though and re-flash it to be a photorpad or something making it just a code based problem if someone was comfortable with using orwl as the base.

**Comment by andrewdavidwong** [comment]
Feel free to repost on qubes-devel if you want to discuss it now.

**Comment by frozentime345** [comment]
Kk ill do that soon, can I post feature requests there as well or is it preferred here? Mostly I dislike github for not auto updating without refreshing the page, not sure if google groups is better.

**Comment by andrewdavidwong** [comment]
> Kk ill do that soon, can I post feature requests there as well or is it preferred here?

The reason that it would be appropriate to discuss this on qubes-devel is because the feasibility of the feature request is uncertain and needs to be clarified via discussion before any work can begin. More information:

https://www.qubes-os.org/support/#qubes-devel

> Mostly I dislike github for not auto updating without refreshing the page(google groups appears slightly better at this).

Many people use our lists as [mailing lists rather than forums](https://www.qubes-os.org/support/#mailing-lists-vs-forums), so this consideration is not relevant to everyone. It may be up to your personal preference.

**Comment by frozentime345** [comment]
So I tried posting my message from google groups but nothing happened, was it caught by the spam filter or something, is that even possible from google groups? I really want to edit it lol so I've been checking the group regularly =p.
